src/GLideNHQ/TxTexCache.h

```diff
@@ -36,7 +36,7 @@ class TxTexCache : public TxCache
 
 public:
   ~TxTexCache();
-  TxTexCache(int options, int cachesize, const wchar_t *cachePath, const wchar_t *ident, dispInfoFuncExt callback);
+  TxTexCache(int options, int cachesize, const wchar_t *cachePath, const wchar_t *fullTexPath, const wchar_t *ident, dispInfoFuncExt callback);
```

Comment:
TxTexCache does not need fullTexPath.
All changes in TxTexCache.h  and TxTexCache.cpp can be dropped.